packages/sit-onyx/src/components/OnyxInput/OnyxInput.vue

```diff
@@ -92,6 +95,25 @@ const errorClass = useErrorClass(showError);
             :maxlength="props.maxlength"
             :aria-label="props.hideLabel ? props.label : undefined"
             :title="props.hideLabel ? props.label : undefined"
+            @focus="isFocused = true"
+            @blur="isFocused = false"
+          />
+          <button
+            v-if="
+              !hideClearIcon && isFocused && value !== '' && value !== undefined && value !== null
+            "
+            type="button"
+            class="icon"
```

Comment:
onyx styles are global so they should have a unique global class like "onyx-input__icon"

Reply:
changed it